gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -341,6 +360,19 @@ private void onApplyWindowInsets(WindowInsetsCompat insets) {
     updateBottomSheetPeekHeight(insets);
   }
 
+  private void onAddPolygon(Point point) {
```

Comment:
This combination of name + args imply that adding is a polygon only requires a single point, when in fact this is called when adding a _vertex to a polygon_ when in add polygon mode. Consider renaming something like `onAddVertex`.